src/app/static/src/tests/root/actions.test.ts

```diff
@@ -319,14 +321,16 @@ describe("root actions", () => {
 
         const expectedError = {error: "OTHER_ERROR", detail: "TestError"};
         expect(commit.mock.calls[0][0]).toEqual({payload: expectedError, type: RootMutation.ErrorReportError});
+        expect(dispatch.mock.calls.length).toBe(0)
         expect(mockAxios.history.post.length).toEqual(1)
         expect(mockAxios.history.post[0].url).toEqual(url)
 
         const expected = {
             email: "some.user@example.com",
-            country: "",
+            country: "no associated country",
```

Comment:
So this test is testing both that the behaviour when the POST fails is as expected, and that the error report defaults are populated? I'd prefer it if that was reflected in the test name or tested in separate tests.

Reply:
+1 for this being 2 tests please!
* "can return error when error report fails"
* "sends default values when country, project or jobId are missing"